railties/test/generators/authentication_generator_test.rb

```diff
@@ -52,6 +52,10 @@ def test_authentication_generator
 
     assert_file "test/models/user_test.rb"
     assert_file "test/fixtures/users.yml"
+
+    assert_file "test/test_helper.rb" do |content|
```

Comment:
In order for this to work in this test, you'll need something like this in the `setup` method:

```patch
diff --git a/railties/test/generators/authentication_generator_test.rb b/railties/test/generators/authentication_generator_test.rb
index 93653e8c..4fbabd58 100644
--- a/railties/test/generators/authentication_generator_test.rb
+++ b/railties/test/generators/authentication_generator_test.rb
@@ -13,6 +13,14 @@ def setup
       class ApplicationController < ActionController::Base
       end
     RUBY
+    FileUtils.mkdir_p("#{destination_root}/test")
+    File.write("#{destination_root}/test/test_helper.rb", <<~RUBY)
+      require "rails/test_help"
+      module ActiveSupport
+        class TestCase
+        end
+      end
+    RUBY

     copy_gemfile

```